server/aws-lsp-partiql/src/server/syntax-highlighting/parser-tokens.ts

```diff
@@ -0,0 +1,195 @@
+import Parser from 'web-tree-sitter'
```

Comment:
I think you forgot to add `web-tree-sitter` as a dependency to the package.json of this server.

Reply:
I believe `web-tree-sitter` package has already be centrally managed over [here](https://github.com/aws/language-servers/blob/main/package-lock.json#L16074).

Reply:
Thanks for catching that! Added it.